api/src/main/java/org/pmiops/workbench/cdr/MySQLDialect.java

```diff
@@ -3,7 +3,7 @@
 import org.hibernate.dialect.function.SQLFunctionTemplate;
 import org.hibernate.type.StandardBasicTypes;
 
-public class MySQLDialect extends org.hibernate.dialect.MySQL57InnoDBDialect {
+public class MySQLDialect extends org.hibernate.dialect.MySQL5Dialect {
```

Comment:
Is `hibernate.dialect.storage_engine=innodb` necessary here too?

https://docs.jboss.org/hibernate/orm/5.4/javadocs/org/hibernate/dialect/MySQL57InnoDBDialect.html
https://in.relation.to/2017/02/20/mysql-dialect-refactoring/

Reply:
I wasn't sure, but I think our tests would fail if it were necessary.